src/wallet/rpc/spend.cpp

```diff
@@ -98,7 +98,7 @@ std::set<int> InterpretSubtractFeeFromOutputInstructions(const UniValue& sffo_in
 static UniValue FinishTransaction(const std::shared_ptr<CWallet> pwallet, const UniValue& options, const CMutableTransaction& rawTx)
 {
     // Make a blank psbt
-    PartiallySignedTransaction psbtx(rawTx);
+    PartiallySignedTransaction psbtx(rawTx, /*version=*/2);
```

Comment:
Whitespace is different here compared to below (`mtx, /* version=*/ 2`). I don't know if you guys worry about that sort of thing.

Reply:
Fixed the whitespace to be consistent with our style guidelines